network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -743,13 +745,36 @@ def get_voting_json(self):
         votes = product.votes.get_votes()
         data = {
             'creepiness': {
-                'vote_breakdown':  {k: v for (k, v) in enumerate(votes)},
+                'vote_breakdown': {k: v for (k, v) in enumerate(votes)},
                 'average': product.creepiness
             },
             'total': product.total_vote_count
         }
         return json.dumps(data)
 
+    # See package docs for `get_meta_*` methods: https://pypi.org/project/wagtail-metadata/
+    def get_meta_title(self):
+        return f"*Privacy Not Included review: {self.title}"
+
+    def get_meta_description(self):
+        if self.search_description:
+            return self.search_description
+
+        soup = BeautifulSoup(self.blurb, "html.parser")
+        first_paragraph = soup.find("p")
+        if first_paragraph:
+            return first_paragraph.text
+
+        return super().get_meta_description()
```

Comment:
The parent classes method also runs
```if self.search_description:
        return self.search_description
```
But the small repetition seemed better than copying the code